lib/Interpreter/CppInterOp.cpp

```diff
@@ -1600,7 +1600,12 @@ namespace Cpp {
                             PrintingPolicy Policy) {
       //TODO: Implement cling desugaring from utils::AST
       //      cling::utils::Transform::GetPartiallyDesugaredType()
-      QT = QT.getDesugaredType(C);
+      if (!QT->isTypedefNameType() || QT->isBuiltinType())
+        QT = QT.getDesugaredType(C);
+#if CLANG_VERSION_MAJOR > 16
+      Policy.SuppressElaboration = true;
+#endif
```

Comment:
The `SuppressElaboration` Printing Policy is not part of `clang <= 16`, therefor CI fails.